lib/init.c

```diff
@@ -11,24 +11,30 @@ int metal_init(const struct metal_init_params *params)
 {
 	int error = 0;
 
-	memset(&_metal, 0, sizeof(_metal));
+	if (_metal.common.ref_count++ == 0) {
+		memset(&_metal.common + 1, 0,
+		       sizeof(_metal) - sizeof(_metal.common));
```

Comment:
can be write on a single line ( 100 char/line)

Reply:
Done.